### ISSUE TITLE
[5.x]: Docs and example templates - add deprecation warning for subscriptions (stripe)

### ISSUE THREAD

**Issue by mandrasch** [craftcms/commerce#3682]
### Description

[src/shop/plans/index.twig](https://github.com/craftcms/commerce/blob/5.x/example-templates/src/shop/plans/index.twig)  has no warning about the fact that subscriptions should be migrated to https://github.com/craftcms/stripe

The docs (https://craftcms.com/docs/commerce/5.x/system/subscriptions.html) have no warning as well.

I just discovered the new recommendation to use https://github.com/craftcms/stripe (instead of https://github.com/craftcms/commerce-stripe) for new projects by accident in Craft discord - after I invested some time to learn about the (old) native subscriptions via example templates and this [craft quest course](https://craftquest.io/courses/craft-commerce-subscriptions). 

### Steps to reproduce

1. Add example templates

### Expected behavior

Warning about deprecation of native Craft Commerce Subscriptions in future, link to migration information: https://github.com/craftcms/stripe?tab=readme-ov-file#migrating-from-craft-commerce

### Actual behavior

No warning

### Craft CMS version

5.x

### Craft Commerce version

5.x

### Installed plugins and versions

-

**Comment by lukeholder** [comment]
@mandrasch Thanks for that feedback. We will be updating the docs and information soon. In the meantime - we will continue support subscriptions in Commerce, and it won't be removed outside of advanced public warnings.

**Comment by mandrasch** [comment]
Thanks very much for quick response, @lukeholder! 

But it is still correct that the craft team recommends using the new https://github.com/craftcms/stripe subscriptions for new projects, right? 

We're currently working on a new v5 commerce project under immense time pressure and it's hard to get going / start with productive coding here. I added a question here https://github.com/craftcms/stripe/issues/25 and we're using REST API / headless approach - which is our challenge/responsibility of course. But to be honest the whole developer experience is a bit disappointing in general currently 🙁 We as agency developers (most likely) get the blame and the heat for these kind of delays. I know this is the developer game, but I can only give you the honest feedback that this is not a good experience for new commerce projects with subscriptions right now. Hope this can be resolved for other devs in future.

Update: We decided to stick to the current "old" way for now.
